src/main/webapp/index.html

```diff
@@ -47,7 +47,7 @@
 
 <div class="navbar navbar-fixed-top">
     <div class="navbar-inner">
-        <div class="userName-top"><span id="user"></span> | <a href="/logout" id="logout-link">Log out</a></div>
+        <div class="userName-top"><span id="user"></span> | <a href="/v1/logout" id="logout-link">Log out</a></div>
```

Comment:
this won't work with the portable ui fix #11 will it?  (this is definitely already an improvement but if there's a further quick fix here or in #11 that would be nice)